src/test/java/lotto/model/LotteryTest.java

```diff
@@ -54,25 +54,102 @@ public class LotteryTest {
                 LotteryNumber.of(42),
                 LotteryNumber.of(43)));
         Set<Integer> winNumbers = Set.of(41, 1, 13, 27, 42, 43);
-        int count = lotteryTicket.matchCount(new WinNumbers(winNumbers));
+        double count = lotteryTicket.matchCount(new WinNumbers(winNumbers));
         assertThat(count).isEqualTo(3);
     }
 
     @Test
-    void 총_당첨_금액() {
-        Map<Win, Integer> winTotal = new EnumMap<>(Win.class);
-        winTotal.put(Win.WIN_3, 3);
-        winTotal.put(Win.WIN_4, 4);
-        winTotal.put(Win.WIN_5, 0);
-        winTotal.put(Win.WIN_6, 1);
-        assertThat(Win.totalMoney(winTotal)).isEqualTo(2000215000);
+    void 우승_통계_계산() {
+        LotteryTicket win3 = new LotteryTicket(Set.of(
```

Comment:
```suggestion
        LotteryTicket win3 = LotteryTicket.newInstance(Set.of(8,21,23,41,42,43));
```
이런식으로도 리팩토링이 가능할 것 같네요

Reply:
@bahar-j  
아 이부분에 대해 질문하셨었군요.. 메일 notify가 너무 많아서 놓쳤네요 ㅠㅠ 
이 부분은 정적 팩토리 메서드에 대해 언급한것입니다 ㅎㅎ 다음 링크를 참고해보시면 좋을 것 같아요!
https://catsbi.oopy.io/d7f3a636-b613-453b-91c7-655d71fda2b1#b870b405-098e-4bea-9e90-96f151d03fe3